.travis.yml

```diff
@@ -18,75 +18,64 @@ cache:
 
 install:
 - grep -Erl git-lfs\|couchdb /etc/apt | sudo xargs rm -v
+- pip install -U pip==20.2.1
```

Comment:
`pip >= 20.2` is what triggers the UnicodeDecodeError you had to work around in ansible_install.py

Reply:
Ok, nice find! I'm in favor of passing the encoding on command install rather than pinning to an older version of pip, so the fix isn't lost in the future. I'm imagining a scenario where a new pip version comes along that we want, and then we try and upgrade to it and run into the potential UnicodeDecodeError again, but forget how to resolve it. For that reason, I'm inclined to keep the encoding as-is in ansible-install.py. WDYT?